packages/financial-templates-lib/src/logger/SpyTransport.js

```diff
@@ -22,29 +21,27 @@ class SpyTransport extends Transport {
 // Helper function used by unit tests to check if the last message sent to winston contains a particular string value.
 // Caller feeds in the spy instance and the value to check.
 function lastSpyLogIncludes(spy, value) {
-  // Sinon's getCall(n) function returns values sent in the nth (zero-indexed) call to the spy. Check both the mrkdown and message sent.
-  const lastReturnedArgMrkdwn = spy.getCall(-1).lastArg.mrkdwn.toString();
-  const lastReturnedArgMessage = spy.getCall(-1).lastArg.message.toString();
-  return lastReturnedArgMrkdwn.indexOf(value) !== -1 || lastReturnedArgMessage.indexOf(value) !== -1;
+  // Sinon's getCall(n) function returns values sent in the nth (zero-indexed) call to the spy. Flatten the whole object
+  // and any log messages included and check if the provided value is within the object.
+  const lastLogMessage = JSON.stringify([
+    spy.getCall(-1).lastArg,
```

Comment:
Won't this first argument include the error message? Why do you have to have a second element just for the error message?

Reply:
it does include it but the error message is not a string. its a Javascript error object. We need to pull out the message from the `lastArg` to effectively query the `message` prop. I'm open to other ways of doing this

Reply:
Ahhh,  no this makes sense -- the way you have it SGTM!